news/2020-02-18-mappers-report-february.md

```diff
@@ -0,0 +1,178 @@
+
+### General
+
+- [**Custom Difficulty Naming Updates**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.1): 
+A description has been added to the rule “A beatmapset's custom difficulty naming must follow a common theme or pattern related to the song…” to clarify its meaning.
+- [**Spread and Length Requierements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.1): The requirement that drain time must be 80% of play time has been removed from the point that “Difficulties lower than the highest difficulty can use their play time as a metric instead of drain time”.
+- [**Sped Up Ver. Markers**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.2): Songs that have been edited to have a higher tempo are now required to use a (Sped Up Ver.) marker in the title field. For songs where the genre is Techno, Trance, Dance, or other similar genres, (Nightcore Mix) will be used instead.
+- [**Hitsound File Length**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.4): The minimum length for hitsound files has been reduced from 100ms to 25ms, after testing showed that 25ms hitsound files will behave correctly in-game.
+- [**Cutting Songs Guideline Changes**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.4): Cuts are required to maintain the general impression and intensity of the full song, however the guideline about the cuts having to be 1:30 or above in length has been removed. 
+- [**Background Requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.5): New minimum width and height sizes have been added in addition to the already present maximum width and height rules. The new minimum sizes are 160 pixels wide and 120 pixels high. 
+- [**Background Quality Guideline**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#guidelines.5): A new guideline has been added specifying that background images must be of reasonable quality.
+
+### osu!taiko
+
+- **Drum Sampleset**: A rule was removed which forbade the use of the drum sampleset. Previously, the game had no drum sample set support for osu!taiko, but this was fixed and samples added. Thus, the rule has been removed as the sampleset is now usable.
+
+### osu!catch
+
+- [**HP Difficulty Guidelines**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!catch#difficulty-specific): The difficulty specific guidelines for HP settings have been adjusted for Cup to Overdose to be in line with community standards.
+
+### osu!mania
+
+- [**Hitsound Additions Requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria#rules.4): Following this change, osu!mania beatmapsets containing only difficulties of Insane or above do not require hitsound additions. 
+
+- [**Difficulty name requirements**](https://osu.ppy.sh/help/wiki/Ranking_Criteria/osu!mania#rules):
+A new specification has been added that sets with a single key mode must not use the key mode in the difficulty name.
+
```

Comment:
celerih says points should probably be spaced here as in the general section (i.e. no newlines between each)